src/ContainerInstance/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,70 @@
+Invoke-LiveTestScenario -Name "Create ContainerGroup" -Description "Test New-AzContainerGroup" -ScenarioScript `
+{
+    param ($rg)
+    $rgName = $rg.ResourceGroupName
+    $containerName = New-LiveTestResourceName
+    $cgName = New-LiveTestResourceName
+    $cgLocation = "westus"
+    $container = New-AzContainerInstanceObject -Name $containerName -Image alpine
+    $actual = New-AzContainerGroup -ResourceGroupName $rgName -Name $cgName -Location $cgLocation -Container $container 
+    Assert-AreEqual $cgName $actual.Name
+    Assert-AreEqual $cgLocation $actual.Location
+}
+
+Invoke-LiveTestScenario -Name "List ContainerGroup" -Description "Test listing ContainerGroup" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $containerName = New-LiveTestResourceName
+    $cgName = New-LiveTestResourceName
+    $cgLocation = "westus"
+    $container = New-AzContainerInstanceObject -Name $containerName -Image alpine
+    $null = New-AzContainerGroup -ResourceGroupName $rgName -Name $cgName -Location $cgLocation -Container $container 
+    $actual = Get-AzContainerGroup -ResourceGroupName $rgName
+    Assert-AreEqual 1 $actual.Count
```

Comment:
This assertion may have potential risk due to retry mechanism. If Get- fails, then the next time the resource group will host 2 container instances. The reason is that retry won't create new resource group but create under the same resource group where the error is thrown.